task/remote-scp/0.1/remote-scp.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: remote-scp
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
```

Comment:
latest version of pipelines is 0.59, since this is a new task, perhaps we can keep a near latest version for this.